src/HomePage/components/MovieCard/MovieCard.tsx

```diff
@@ -0,0 +1,40 @@
+import React, { useState } from 'react';
+import Popup from "../Popup/Popup";
+import './MovieCard.css'
+import '../Popup/Popup.css'
+
+interface MovieCardInterface {
+    title: string,
+    popularity: number
+}
+
+export function MovieCard(props: MovieCardInterface) {
+    const [isOpen, setIsOpen] = useState(false);
+    const [isEdited, setIsEdited] = useState(false);
+    const [isDeleted, setIsDeleted] = useState(false);
+
+    const togglePopup = () => {
+        setIsOpen(!isOpen);
+    }
+
+    const toggleEdit = () => {
+        setIsEdited(!isEdited);
+    }
+
+    const toggleDelete = () => {
+        setIsDeleted(!isDeleted);
+    }
```

Comment:
You have the same approach repeated thrice - so I suggest you to create custom hook in separate file

```
const useTogglerState = (initialValue?: boolean) => {
   const [value, setValue] = useState(initialValue);
   const toggleValue = () => setValue(current => !current);

   return [value, toggleValue];
}
```

And use it instead